orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/RestrictExecutionDuringTimeWindow.groovy

```diff
@@ -41,8 +44,28 @@ class RestrictExecutionDuringTimeWindow implements StageDefinitionBuilder {
   public static final String TYPE = "restrictExecutionDuringTimeWindow"
 
   @Override
+  @CompileStatic(TypeCheckingMode.SKIP)
   void taskGraph(Stage stage, TaskNode.Builder builder) {
     builder.withTask("suspendExecutionDuringTimeWindow", SuspendExecutionDuringTimeWindowTask)
+    if (stage.context.restrictedExecutionWindow?.splay?.enabled && stage.context.restrictedExecutionWindow?.splay?.maxDelay) {
```

Comment:
You can skip the null checks on the second part, i.e. `stage.context.restrictedExecutionWindow.splay.maxDelay` should be fine.